python/common/business-registry-model/src/business_model_migrations/alembic.ini

```diff
@@ -2,7 +2,7 @@
 
 [alembic]
 # template used to generate migration files
-# file_template = %%(rev)s_%%(slug)s
+file_template = %%(year)d%%(month).2d%%(day).2d_%%(hour).2d%%(minute).2d%%(second).2d_%%(rev)s_%%(slug)s
```

Comment:
NOTE: this automatically adds the date to the beginning of the migration file name. I've found this helpful in other APIs, but lmk what you think. It looks like the existing migrations are ordered with a number in front of them, but I think that was a manual edit to the names of the files?